qa/rpc-tests/test_framework.py

```diff
@@ -128,10 +128,14 @@ def main(self):
             print("Unexpected exception caught during testing: "+str(e))
             traceback.print_tb(sys.exc_info()[2])
 
-        if not self.options.nocleanup:
-            print("Cleaning up")
+        try:
             stop_nodes(self.nodes)
             wait_bitcoinds()
+        except:
+            pass
```

Comment:
I might be wrong, but it looks like there doesn't seem to be an exception raised to begin with. I removed the catch completely in the updated version.